api/proto/openyolo.proto

```diff
@@ -21,9 +21,8 @@ package org.openyolo;
 option java_package = "org.openyolo.proto";
 
 message CredentialRetrieveRequest {
-    repeated string       authDomains        = 1; // at least one required
-    repeated string       authMethods        = 2; // at least one required
-    repeated KeyValuePair additionalParams   = 3;
+    repeated string       authMethods        = 1; // at least one required
```

Comment:
be careful when modifying protos - renumbering fields can horribly break things. Leave the original numbers the same, and put a comment in this file that field 1 is deprecated. It's not super important just now when there are no production apps using openyolo, but we should start treating it like it is.